content/blog/2019/secure-egress-control-istio/index.md

```diff
@@ -0,0 +1,324 @@
+---
+title: Istio as a transparent DNS-aware Kubernetes-aware Egress Firewall
+subtitle: Secure Egress Traffic Control in Istio
+publishdate: 2019-02-04
+attribution: Vadim Eisenberg (IBM)
+weight: 70
```

Comment:
In your various upcoming blog posts, please remove the weight: field. We don't need these anymore, the blogs get auto-sorted by publication date.

Thanks.

Reply:
Done in https://github.com/istio/istio.io/pull/3179/commits/5d4018ebb950a52ac562b1f0761251ba77d445c3